### DISCUSSION TITLE
webdav 电脑和手机间同步失败 invalid response： 404 Not Found

### DISCUSSION THREAD

**Discussion by ArMorind** [remotely-save/remotely-save#763] [Q&A]
在windows和安卓检查连接时都显示可以连上服务器，服务器端也可以看到所有文件，但是电脑和手机的文件互相不能查看

## 这是服务器端Vault内容
![image](https://github.com/user-attachments/assets/b415fca6-40fb-4d44-a355-e6ef61b7b500)

## 电脑端报错（报错内容均为手机写的文件）
![image](https://github.com/user-attachments/assets/f16f2890-62bf-456a-97eb-40246413a102)

## 电脑端同步后Vault目录
![image](https://github.com/user-attachments/assets/f1ed14e9-2387-4f6d-ae48-3bedd9acb3ac)

## 手机端目录
![image](https://github.com/user-attachments/assets/7e24af4c-82d5-4f6d-a485-d7d1815ffb6b)

##  手机端报错（报错内容均为电脑生成文件）
![image](https://github.com/user-attachments/assets/75c3c41b-3bbd-44f1-9f7a-a95b8a3907dd)

**Comment by fyears** [comment]
确认没有权限问题？这个账号可以下载？

服务器是怎么搭建的？有可复现步骤吗？

**Reply by ArMorind** [reply]
服务器是用的自建的windows主机用IIS设置的webdav，读写权限都开了，
![image](https://github.com/user-attachments/assets/7c07df40-3b75-40da-8257-c1ccc0e40865)

用另一台电脑使用相同账户在浏览器打开没有问题，可以显示内容
![image](https://github.com/user-attachments/assets/7697ffdc-7f73-4c5a-97aa-20f36178b1a3)


没理解你说的这个账号可以下载指的是什么，复现步骤目前我没有别的服务器可以测试，基本上来说只是用IIS设置了最基本的webdav功能，然后remotelysave用的服务器管理员账户直接登录webdav网址，测试连接成功后尝试同步文件时就遇到了这些问题。
![image](https://github.com/user-attachments/assets/8b60ffae-ce8d-4eaf-a06d-a8d476c6d1a0)

**Comment by fyears** [comment]
请使用 0.5.16 的 check connection 再试一下。加了一些检查连接的错误提示

**Comment by ArMorind** [comment]
抱歉之前忘了测试，目前问题依旧存在

**Reply by ArMorind** [reply]
> 抱歉之前忘了测试，目前问题依旧存在

![image](https://github.com/user-attachments/assets/0f84bf1c-38ff-4c59-867e-0efed3e323ac)

**Reply by ArMorind** [reply]
服务器地址在浏览器中还是可以正常访问 但是obsidian反馈是404

**Reply by fyears** [reply]
服务器地址会不会是漏了子文件夹，最后加上和不加 / 都试一下呢

**Reply by ArMorind** [reply]
> 服务器地址会不会是漏了子文件夹，最后加上和不加 / 都试一下呢

目前我的vault就是挂在服务器根目录下，尝试加上了vault文件夹名字的路径以及加上和不加/都没有区别

**Reply by ArMorind** [reply]
> 服务器地址会不会是漏了子文件夹，最后加上和不加 / 都试一下呢

![image](https://github.com/user-attachments/assets/8d4f3106-0476-41ad-9373-e7055f681723)
服务器端根目录如下 GuGuVault为当前我在使用的vault名称
![image](https://github.com/user-attachments/assets/812e515c-cf4c-49d4-9925-9a9024378573)
尝试加上vault名称后创建了另一个GuGuVault文件夹

**Reply by LazyHoboKilla** [reply]
Did you find a solution? i am experiencing the same issue

**Comment by cremecho** [comment]
请问题主解决了吗，我遇到了同样的问题
